LDMP/calculate_ldn.py

```diff
@@ -302,7 +302,7 @@ def btn_calculate(self):
                 year_final,
                 'legend_nesting':
                 LCLegendNesting.Schema().dump(
-                    self.lc_setup_widget.aggregation_dialog.nesting
+                    self.lc_setup_widget.aggregation_dialog._nesting
```

Comment:
Should now be referencing `lc_setup.get_lc_nesting()`?

Reply:
See previous comment...most likely have been as a result of auto-refactoring which was only in `lc_setup.py`